packages/playground/blueprints/src/lib/steps/handlers.ts

```diff
@@ -3,6 +3,7 @@ export { activateTheme } from './activate-theme';
 export { applyWordPressPatches } from './apply-wordpress-patches';
 export { runPHP } from './run-php';
 export { runPHPWithOptions } from './run-php-with-options';
+export { runSqlFile } from './run-sql-file';
```

Comment:
These changes seem to belong to https://github.com/WordPress/wordpress-playground/pull/745 – once this is approved let's hold on with merging until #745 lands and this is rebased.

Reply:
@adamziel This has been cleaned up. That file is no longer relevant to ANY pull request.